presto-docs/src/main/sphinx/release/release-0.226.rst

```diff
@@ -0,0 +1,21 @@
+=============
+Release 0.226
+=============
+
+General Changes
+_______________
+* Add peak running task count to statistics field of QueryCompletedEvent
+* Fix ST_Buffer for tiny geometries (#13194)
+* Fix ST_Centroid for tiny geometries (#10629)
+* Fix bug that causes failure when reading unpartitioned table from some connectors with session property ``grouped_execution_for_eligible_table_scans`` turned on.
+* Reuse compression buffer in PagesSerde when serializing pages.
+* Improve memory tracking for geospatial query indexing and prevent worker node down due to out of memory
+* Use Hilbert Packed RTree (Flatbush) instead of the JTS STR RTree for spatial joins.  The Flatbush is a flattened tree that uses vastly fewer heap allocations, and has faster build and query times.
```

Comment:
nit: perhaps, `Reduce memory usage of spatial joins by using Hilbert Packed RTree (Flatbush) instead of the JTS STR RTree. The Flatbush is a flattened tree that uses vastly fewer heap allocations, and has faster build and query times.` CC: @jagill